<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMYWidget</class>
 <widget class="QWidget" name="QMYWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMYWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>刻晴</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox1"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAgeAdd1">
       <property name="text">
        <string>Age++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMetaInfo">
       <property name="text">
        <string>MetaObjectInfo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label1_2">
       <property name="text">
        <string>申鹤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox2"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAgeAdd2">
       <property name="text">
        <string>Age++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClean">
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClean</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
